HW_classes.js

```diff
@@ -0,0 +1,49 @@
+// class Human {
+//   constructor(name, age) {
+//     this.name = name;
+//     this.age = age;
+//   }
+// }
+// Human.prototype.ageOneYear = function() {
+//   this.age++;
+//   return `${this.age}`;
+// };
+// Human.prototype.eating = function() {
+//   console.log("mmm, mmm, mmm, I'm love'n it");
+// };
```

Comment:
Really great job! Alternatively, you could have placed these functions within the class using the syntax: 
```
ageOneYear() {
  return this.age++
}
```
Same goes for your `eating` function!